### PR TITLE
fixing link version bug

### LINKED ISSUE #179: Links on index page to Concepts, Architecture,... are broken
https://keda.sh/ ->
Links in the main section on the index page, (white buttons) are broken.
`Concepts, Deploying KEDA, Architecture, Scalers, Blog`

Probably because of migration to versioned docs.

### PR DESCRIPTION
**Pull request by jeffhollan**
Resolves #179

Was missing an else so the logic made it so it would always overwrite the regex (or never set the variable if it wasn't a doc)

### CODE REVIEW

**Comment by lucperkins** [comment]
This is not necessary. You can use `/docs/latest/*` to make sure that this works on all versions.

**Comment by lucperkins** [comment]
Ah okay, I see the issue now. Disregard my comment in the issue.

File: config.toml [hunk 1]
```diff
@@ -102,7 +102,7 @@ url = "https://kubernetes.slack.com/messages/CKZJ36A5D"
 
 # Main page hero buttons
 [[menu.buttons]]
-url = "/docs/concepts"
+URL = "/docs/concepts"
```
Comment (lucperkins) [inline config.toml:105]: Uppercasing this has no effect AFAIK but best to keep these consistent

File: config.toml [hunk 2]
```diff
@@ -133,7 +133,7 @@ weight = 5
 
 # Navbar and footer menu
 [[menu.docs]]
-url = "/docs/"
+url = "/docs"
```
Comment (jeffhollan) [inline config.toml:136]:
```suggestion
url = "/docs/"
```

**Comment by jeffhollan** [comment]
Thanks @tomkerkhove - made the changes

**Pull request merged**